.github/workflows/third-party.yaml

```diff
@@ -36,6 +36,8 @@ jobs:
           apk update
           apk add bash curl findutils gh git go nodejs perl upx xz yara-x-compat
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - name: Trust repository
+        run: git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
`$GITHUB_WORKSPACE` should probably be quoted; I wouldn't expect there to be a space or other non-ascii in the name, but it's good practice to do so anyway.

Reply:
Good call. I'll run through all of the references and update them.

Reply:
Updated in [`1fbbb0f` (#753)](https://github.com/chainguard-dev/malcontent/pull/753/commits/1fbbb0f9541c20aee5ad3a977e3e1beac39cf18d).